src/mono/wasm/templates/templates/console/README.md

```diff
@@ -12,10 +12,10 @@ After building the application, the result is in the `bin/$(Configuration)/net7.
 
 ## Run
 
-You can build the applcation from Visual Studio or by dotnet cli
+You can build the application from Visual Studio or by dotnet cli
 
 ```
-dotnet run -c Debug/Release -r browser-wasm -h=nodejs
+dotnet run -c Debug/Release -r browser-wasm -- -h=node
```

Comment:
which part of the app host would process things after `--` ? I thought that should be passed to the application code's `Main()` @radical